lib/cisco_node_utils/cmd_ref/pim_rp_address.yaml

```diff
@@ -0,0 +1,32 @@
+# pim_rp_address
+---
+_template:
+  config_get : "show run pim all"
+  config_get_token :
+    - '/^vrf context <vrf>$/'
+  config_set :
+    - "vrf context <vrf>"
+
+all_group_lists:
+  multiple:
+  config_get_token_append : '/^ip pim rp-address ([0-9\.]+) group-list (\S+)/'
+
+all_rp_addresses:
+  multiple:
+  config_get_token_append: '/^ip pim rp-address (.*)$/'
+
+feature:
+  config_get_token_append: '/^feature pim$/'
+  config_set: "feature pim"
```

Comment:
No need to specify this here. You can use `Pim.feature` from any provider instead of duplicating it here.